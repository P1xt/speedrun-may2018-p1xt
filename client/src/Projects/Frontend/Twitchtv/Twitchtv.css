.Twitchtv {
  text-align: left;
  margin-top: 40px; }
  .Twitchtv > h1 {
    text-transform: none; }
