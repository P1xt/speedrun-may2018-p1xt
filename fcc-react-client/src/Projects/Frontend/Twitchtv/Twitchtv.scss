.Twitchtv {
  margin-top: 40px;

  h2 {
    text-align: center;
    text-transform: none;
    margin-bottom: 20px;
  }
  img {
    width: 50px;
    border-radius: 10px;
    margin-right: 40px;
  }
  span {
    margin-top: 10px;
    float: right;
  }
}
