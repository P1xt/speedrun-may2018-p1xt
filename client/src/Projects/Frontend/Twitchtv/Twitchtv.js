import React from "react";
import { Container, Row, Col } from "reactstrap";

import "./Twitchtv.css";

const Twitchtv = () => {
  return (
    <div className="Twitchtv">
      <Container>
        <Row>
          <Col>
            <h1>Twitchtv</h1>
          </Col>
        </Row>
      </Container>
    </div>
  );
};

export default Twitchtv;
