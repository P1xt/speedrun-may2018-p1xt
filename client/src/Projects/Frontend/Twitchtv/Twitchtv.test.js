import React from "react";
import renderer from "react-test-renderer";
import { shallow } from "enzyme";

import Twitchtv from "./Twitchtv";

describe("<Twitchtv />", () => {
  it("renders without crashing", () => {
    shallow(<Twitchtv />);
  });
  it("renders correctly", () => {
    const tree = renderer.create(<Twitchtv />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
