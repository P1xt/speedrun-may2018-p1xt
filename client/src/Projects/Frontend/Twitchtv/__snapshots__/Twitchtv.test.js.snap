// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Twitchtv /> renders correctly 1`] = `
<div
  className="Twitchtv"
>
  <div
    className="container"
  >
    <div
      className="row"
    >
      <div
        className="col"
      >
        <h1>
          Twitchtv
        </h1>
      </div>
    </div>
  </div>
</div>
`;
